### ISSUE TITLE
Node 14 Upgrade

### ISSUE THREAD

**Issue by BenBoersma** [cerner/terra-ui#343]
# Feature Request

## Description

As a maintainer I would like to upgrade this component to Node 14 (lts/fermium)

So that it can have the latest features and security updates and remain tech current with Node.

We know we're done when:

- [ ] The .nvmrc file is using lts/fermium
- [ ] All tests are passing
- [ ] A PR is merged to the main branch

**Comment by marshalc** [comment]
As a new development team looking to start Terra type development, we very much would like to see this as we've been having no end of problems trying to get the toolkit and related build processes to install locally. Of the two developers here trying to get this going, we've got the following combination of environments:

### Me

-   macOS: 11.2.3
-   node: 15.11.0
-   npm: 7.6.0

### My Colleague

-   windows: 10.0.18363 (Version 1909)
-   node: 14.16.0
-   npm: 6.14.11

My expectation is that if you can get things up to at least the Node 14 level, that should have far fewer issues for myself who tends to stay at the current edge versions.

**Comment by mjhenkes** [comment]
Hi @marshalc, [terra-toolkit](https://github.com/cerner/terra-toolkit-boneyard) only supports node 10 (I know not even close to current 😞 ) due to it's [dependance](https://github.com/cerner/terra-toolkit-boneyard/blob/main/package.json#L98) on wdio-mocha-framework which, in turn, depends on wdio-sync which depends on [fibers v4.0.1](https://github.com/webdriverio-boneyard/wdio-sync/blob/master/package.json#L41). The Fibers versions have a very narrow range of node support. [Fibers 3](https://github.com/laverdet/node-fibers/tree/3.1.1) supports node 8 and 10, [Fibers 4](https://github.com/laverdet/node-fibers/tree/v4.0.3) supports node 10 and 12 and [Fibers 5](https://github.com/laverdet/node-fibers/tree/v5.0.0) supports node 12 and 14. (I'm only specifying the LTS versions of node here, because node does semver [differently](https://nodejs.org/en/about/releases/)). 

The terra-toolkit package is on the verge of being replaced by our suite of config and testing packages documented on [terra-ui](https://engineering.cerner.com/terra-ui/dev_tools). Of particular interest to this issue is the new [terra-functional-testing package](https://engineering.cerner.com/terra-ui/dev_tools/cerner/terra-functional-testing/about) where we have updated to support wdio 6. As part of that update the fibers dependency was updated to version 4. Wdio 7 recently released with fibers 5 support and we [plan](https://github.com/cerner/terra-toolkit/issues/585) to move to that soon so that we can finally get on the most current LTS version of node.

tl;dr
terra-toolkit is stuck on node 10.
You can remove the terra-toolkit dependency and use terra-functional-testing and webpack-config-terra to get to node 12
node 14 support for our developer tools is planned

**Comment by sdadn** [comment]
Resolved by: https://github.com/cerner/terra-ui/pull/447
